contracts/vaults/Claim.sol

```diff
@@ -62,14 +63,27 @@ contract Claim is Base {
         );
     }
 
+    function challengeClaim(uint256 _claimId) external onlyArbitrator {
+        require(claims[_claimId].beneficiary != address(0), "HVE10");
+        claims[_claimId].isChallenged = true;
+    }
+
     /**
     * @notice Approve a claim for a bounty submitted by a committee, and transfer bounty to hacker and committee.
-    * Called only by hats governance.
+    * callable by the  arbitrator, if isChallenged == true
+    * Callable by anyone after challengePeriod is passed and isChallenged == false
     * @param _claimId The claim ID
     */
-    function approveClaim(uint256 _claimId) external onlyOwner nonReentrant {
+    function approveClaim(uint256 _claimId, uint256 _bountyPercentage) external nonReentrant {
         Claim storage claim = claims[_claimId];
+
         require(claim.beneficiary != address(0), "HVE10");
+        require(
+          ((msg.sender == arbitrator && claim.isChallenged) ||
+          (claim.createdAt + challengePeriod < block.timestamp)), "HVEXX"
+        );
+
+        claim.bountyPercentage = _bountyPercentage;
```

Comment:
The legitimacy of this value isn't checked. It can be checked on arbitrator's side but if the caller is a random address then he can choose whatever percentage he wants